application/forms/Command/Object/AcknowledgeProblemForm.php

```diff
@@ -21,6 +22,17 @@ class AcknowledgeProblemForm extends CommandForm
 {
     use Auth;
 
+    public function __construct()
+    {
+        $this->on(self::ON_SUCCESS, function () {
+            Notification::success(tp(
+                'Acknowledged a problem successfully',
```

Comment:
`Acknowledged problem successfully`. Note the missing `a`. It's not just *some* problem, it's *the* problem the user acknowledged just now.